README.md

```diff
@@ -120,32 +74,56 @@ To clone the repository and point the local HEAD to the latest commit in the
 
     git clone --recursive https://github.com/bookbrainz/bookbrainz-site.git
 
-### Installing Packages
-The site depends on a number of node packages which can be installed using npm:
 
-    cd bookbrainz-site/
-    npm install
+## Configuration
 
-This command will also compile the site LESS and JavaScript source files.
+Create a copy of `config/config.json.example` and rename it to `config.json`.
+// Then,
+// edit the values so that they are correct for your environment.
+// You will need to change the PostgreSQL username and password under `database.connection`.
 
-## Configuration
+You will need to set up authentication under `musicbrainz`. To get the `clientID` and `clientSecret` tokens, head to [the MusicBrainz website](https://musicbrainz.org/account/applications) and register a new developer application. You can then copy the tokens for that developer application and paste as strings. 
+
+## Database set-up
+
+When you first start working with BookBrainz, you will need to perform some initialization for PostgreSQL and import the latest BookBrainz
+database dump.
+
+Luckily, we have a script that does just that: from the command line, in the `bookbrainz-site` folder, type and run `./scripts/database-init-docker.sh`.
+Let that run until the command returns.
+
+
+## Running the server
+
+If all went well, you will only need to run `./develop.sh` in the command line from the `bookbrainz-site` folder.
+Press `ctrl+c` to stop the server. The dependencies will continue running in the background.
+
+Wait between 30 and 60 seconds for the server to compile, and point your browser to `localhost:9099`.
```

Comment:
Hmmm, not very portable. A shitty net connection or slow machines blows this out of the water. Sadly, this is very difficult to describe what happens, because the container logs don't really go silent, which is how I would describe when its good to go....